hoot-core-test/src/test/cpp/hoot/core/perty/PertyOpTest.cpp

```diff
@@ -189,10 +189,10 @@ class PertyOpTest : public CppUnit::TestFixture
 
     PertyOp uut;
     uut.setGridSpacing(gridSpacing);
-    uut.setRandomError(0.0, 0.0);
+    //uut.setRandomError(0.0, 0.0);
     uut.setSeed(1);
-    uut.setSystematicError(10.0, 10.0);
-    uut.setCsmParameters(9, 10);
+    //uut.setSystematicError(10.0, 10.0);
+    //uut.setCsmParameters(9, 10);
```

Comment:
These parameters all moved to be defaults, so they don't have to be set in this test anymore?